flake.nix

```diff
@@ -67,7 +67,7 @@
             description = "Initialize and launch the web server";
             exec = ''
               nix eval --json .#schema | jq > webui/schema.json \
-              && if [ ! -p pipe ]; then mkfifo pipe; fi \
+              && chmod +x app.js \
```

Comment:
this is unnecessary, the file can be chmodded and the the permissions carry over git as well